### DISCUSSION TITLE
JavaScript and CSS files not loading, need suggestions!

### DISCUSSION THREAD

**Discussion by philipabala** [community/community#102816] [Programming Help]
### Body

Hello everyone.
So, I have an HTML document with a CSS link and JavaScript in its right places and tags. These files are on the same server as the HTML. But when the page loads, the CSS and JS files won't, and the developer tool message is "Failed to load..." pointing to the CSS and JS files. These used to be working before the OS was upgraded to  Windows 2019. I've made some reading and from it there was  suggestion to add the feature for Static Files in IIS  and also configured the cross-origin setting. The folder where the css and Js files are were already set its security to RXW for the IIS_IUSR. I even tried full control. But nothing works. I've cleared my browser's cache and still the same. Does anyone here experienced similar situation? Please share the solution you made.  I made a temporary solution by adding both the CSS and JS codes into the HTML document. It worked but I know it's not the ideal solution. Please help. Thanks in advance for your suggestions.

BTW, the links work when the href URL are from cdn, google or other external domains.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by phuccoder2000** [comment]


**Reply by philipabala** [reply]
That's right, and that's what I'm trying to know, what area of IIS or Windows I need to configure in order for the files to load. My reading says about adding the Static Files feature in IIS installation. Some other suggestion says to enable the 'Cross-Origin' setup in IIS, and the file or folder where the CSS and js files are, should be configured to allow. at least, read permission. ChatGPT has the same suggestions. I did all that and still not working.
